### DISCUSSION TITLE
Print example

### DISCUSSION THREAD

**Discussion by yngrdyn** [hrynko/vue-pdf-embed#110] [Q&A] [answered]
is there an example around on how to use the print method?
I'm pretty new to vue and I have ran out of ideas on how to use that method

**Comment by hrynko** [comment, marked as answer]
Hi @yngrdyn,

Please check #101. Below you can find a very basic example for Vue 2:

```vue
<template>
  <div>
    <button @click="$refs.myPdf.print()">Print</button>
    <vue-pdf-embed ref="myPdf" source="<PDF_YOURL>" />
  </div>
</template>
```

**Reply by hrynko** [reply]
Please let me know if I can update the README file to make this clearer

**Reply by yngrdyn** [reply]
Yes, I'll uptade the README. Thanks!

**Reply by yngrdyn** [reply]
I opened a PR for this: https://github.com/hrynko/vue-pdf-embed/pull/111. Please take a look
